src/windows/views/tutorial.py

```diff
@@ -208,13 +232,20 @@ def get_object(self, object_id):
         elif object_id == "timeline":
             return self.win.timeline
         elif object_id == "dockVideoContents":
-            return self.win.dockVideoContents
+            return self.win.dockVideo
         elif object_id == "propertyTableView":
             return self.win.propertyTableView
         elif object_id == "transitionsView":
             return self.win.transitionsView
         elif object_id == "effectsView":
             return self.win.effectsView
+        elif object_id == "emojisView":
+            return self.win.emojiListView
+        elif object_id == "actionPlay":
+            play_button = None
+            for toolbutton in self.win.videoToolbar.children():
+                if type(toolbutton) == QToolButton and toolbutton.defaultAction() and toolbutton.defaultAction().objectName() == "actionPlay":
+                    return toolbutton
```

Comment:
This could actually be...
```suggestion
            for w in self.win.actionPlay.associatedWidgets():
                if type(w) == QToolButton:
                    return w
```

Reply:
(The list it returns will be `[the Qtoolbar, the QToolButton]`.)